### ISSUE TITLE
The CDN version leads to a suspicious site

### ISSUE THREAD

**Issue by DannyBen** [mickidum/acc_toolbar#33]
In the CDN version, the "learn more" link leads to webworks.ga/acc_toolbar

**Comment by qdelski** [comment]
Yeah, It is is 2 files: minjs/common.min.js and htmlmin/toolbox.html - no trace in source, just in minified files. It was added to repo about 2017 year.

**Comment by sclibraryservices** [comment]
The [webworks.ga/acc_toolbar](http://webworks.ga/acc_toolbar) link now resolves to a 404 error.
